inferentia/README.md

```diff
@@ -202,4 +227,22 @@ Note:
 1. The `config.pbtxt` and `model.py` should be treated as
 starting point. The users can customize these files as per
 their need.
-2. Triton Inferentia is currently tested with a **single** model. 
\ No newline at end of file
+2. Triton Inferentia is currently tested with a **single** model. 
+
+## Testing Inferentia Setup for Accuracy
+The [qa folder](https://github.com/triton-inference-server/python_backend/tree/main/inferentia/qa)
+contains the necessary files to set up testing with a simple add_sub model. The test
+requires an instance with more than 8 inferentia cores to run, eg:`inf1.6xlarge`.
+start the test, run 
+```
+source <triton path>/python_backend/inferentia/qa/setup_test_enviroment_and_test.sh
+``` 
+where `<triton path>` is usually `/home/ubuntu`/.
+This script will pull the corresponding [server](https://github.com/triton-inference-server/server)

```

Comment:
What does corresponding server mean here? I think by default it picks up 'main' right?
This can be changed by exporting TRITON_SERVER_BRANCH_NAME which you describe below.
Just remove the 'corresponding', and describe in detail in the note.